src/utils/templates.ts

```diff
@@ -28,3 +30,11 @@ export const getUserTempateSelection = async (templatesFolderId: string): Promis
 
     return null;
 }
+
+export const getTemplateFromId = async (templateId: string): Promise<Note | null> => {
+    try {
+        return await joplin.data.get([ "notes", templateId ], { fields: ["id", "title", "body"] });
+    } catch (error) {
```

Comment:
You should log the error here. You don't have to do anything with it, but it will help with troubleshooting later.

Reply:
done